packages/expo-updates/android/src/main/java/expo/modules/updates/statemachine/UpdatesStateContext.kt

```diff
@@ -77,4 +78,37 @@ data class UpdatesStateContext(
       result.putMap("context", contextMap)
       return result
     }
+
+  /**
+   * Creates a Bundle to be returned to JS on a call to nativeStateMachineContext()
+   */
+  val bundle: Bundle
+    get() {
+      return Bundle().apply {
+        this.putBoolean("isUpdateAvailable", isUpdateAvailable)
```

Comment:
optional: no need for explicit `this` in apply block (this is inferred as the target) (ditto for other `this` in this method

Reply:
Done